src/directives/droppable.ts

```diff
@@ -1,32 +1,3 @@
-import {Directive, ElementRef, HostListener, Input, Output, EventEmitter} from '@angular/core';
```

Comment:
Well... This is weird, this line is still in my editor...

OK, I will fix this and btw apply the same fix in the draggable directive.